cmd/lncli/walletrpc_active.go

```diff
@@ -726,6 +734,339 @@ type fundPsbtResponse struct {
 	Locks             []*utxoLease `json:"locks"`
 }
 
+var fundTemplatePsbtCommand = cli.Command{
+	Name: "fundtemplate",
+	Usage: "Fund a Partially Signed Bitcoin Transaction (PSBT) from a " +
+		"template.",
+	ArgsUsage: "[--template_psbt=T | [--outputs=O [--inputs=I]]] " +
+		"[--conf_target=C | --sat_per_vbyte=S] " +
+		"[--change_type=A] [--change_output_index=I]",
+	Description: `
+	The fund command creates a fully populated PSBT that contains enough
+	inputs to fund the outputs specified in either the template.
+
+	The main difference to the 'fund' command is that the template PSBT
+	is allowed to already contain both inputs and outputs and coin selection
+	and fee estimation is still performed.
+
+	If '--inputs' and '--outputs' are provided instead of a template, then
+	those are used to create a new PSBT template.
+
+	The 'outputs' flag decodes addresses and the amount to send respectively
+	in the following JSON format:
+
+	    --outputs='["ExampleAddr:NumCoinsInSatoshis", "SecondAddr:Sats"]'
```

Comment:
took me a while to realise that this command expects the format of the outputs using `"["` brackets instead of `"{"` brackets like the `fund` command 😅 

As is done for the `fund` command, should we also just have an example of what the input list looks like?

Reply:
Yeah, I know... I'd like to change the format of the `fund` command too, but not sure if that would break some existing shell scripts perhaps?

Added the example for the inputs though.